src/workerd/api/basics.c++

```diff
@@ -852,6 +855,9 @@ jsg::Ref<AbortSignal> AbortSignal::deserialize(
     jsg::Lock& js, rpc::SerializationTag tag, jsg::Deserializer& deserializer) {
   auto& handler = KJ_REQUIRE_NONNULL(
       deserializer.getExternalHandler(), "got AbortSignal on non-RPC serialized object?");
+
+  KJ_LOG(WARNING, "An AbortSignal was deserialized from RPC");
```

Comment:
We can probably skip logging deserialization since there would necessarily already be a serialization just before it.